### ISSUE TITLE
CommandableEntity has print statements that does nothing

### ISSUE THREAD

**Issue by willayy** [0-Gixty-0/Grupp-11-OOPP#155]
There are print statements in the Commandable entity moveCommand(), interactCommand(), attackcommand() that does nothing because the game doesn't have a console UI. These should be removed.

**Comment by 0-Gixty-0** [comment]
I'm assuming you mean prints such as this one?

```java
System.out.println(String.format("Objects of type %s cannot move",this.getBody().getClass().getName()));
``` 

I added these so that if we were to accidentally call a moveCommand on an entity that cannot move we would get output in the terminal alerting us to the issue. I experimented with using throw exception but it lead to having a try catch around lots of code and seemed like an unnecessary hassle to deal with.

**Comment by willayy** [comment]
Yep, I figured that was their purpose but do we really need them? I guess we could keep them for debugging but it seems kind of overkill. I will leave the decision up to you, close the issue when you have decided!
